### PR TITLE
[CP-1420] Adding missing tests

### PR DESCRIPTION
**Pull request by igorbezsmertnyi**
Jira: [CP-1420]

**Description**
Part #2 - Adding missing test for settings module

<details>
<summary><b>Screenshots</b></summary>
// put images here
</details>

[CP-1420]: https://appnroll.atlassian.net/browse/CP-1420?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/app/src/settings/reducers/settings.reducer.test.ts [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+// import { AppError } from "App/core/errors"
+import { SettingsEvent } from "App/settings/constants"
+import {
+  settingsReducer,
+  initialState,
+} from "App/settings/reducers/settings.reducer"
+import {
+  fulfilledAction,
+  pendingAction,
+  // rejectedAction,
```
Comment (wojcik92michal) [inline packages/app/src/settings/reducers/settings.reducer.test.ts:15]:
```suggestion
```

File: packages/app/src/settings/actions/toggle-update-available.action.test.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  checkAppUpdateFlowToShow: () => jest.fn(),
+}))
+
+const mockStore = createMockStore([thunk])()
+
+afterEach(() => {
+  jest.clearAllMocks()
+})
+
+test("calls `toggleTethering` action dispatch SettingsEvent.ToggleUpdateAvailable", async () => {
```
Comment (wojcik92michal) [inline packages/app/src/settings/actions/toggle-update-available.action.test.ts:21]: Seems like the test description is invalid - it says something about `toggleTethering`

File: packages/app/src/settings/reducers/settings.reducer.test.ts [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+          updateAvailable: true,
+        },
+        {
+          type: fulfilledAction(SettingsEvent.CheckUpdateAvailable),
+          payload: true,
+        }
+      )
+    ).toEqual({
+      ...initialState,
+      updateAvailable: undefined,
```
Comment (wojcik92michal) [inline packages/app/src/settings/reducers/settings.reducer.test.ts:199]: `undefined`? Is it ok?

**Review by wojcik92michal — APPROVED** [review]

File: packages/app/src/settings/reducers/settings.reducer.test.ts (hunk 3 above)
Comment (igorbezsmertnyi) [reply packages/app/src/settings/reducers/settings.reducer.test.ts:199]: yep, the MC check if the value is `undefined` before. It's mean's that the update checking isn't initializing yet

File: packages/app/src/settings/actions/toggle-update-available.action.test.ts (hunk 2 above)
Comment (igorbezsmertnyi) [reply packages/app/src/settings/actions/toggle-update-available.action.test.ts]: https://github.com/mudita/mudita-center/pull/842/commits/706d90599383cfd7803a08c6fac03dd0e2826dc1

File: packages/app/src/settings/reducers/settings.reducer.test.ts (hunk 1 above)
Comment (igorbezsmertnyi) [reply packages/app/src/settings/reducers/settings.reducer.test.ts]: https://github.com/mudita/mudita-center/pull/842/commits/706d90599383cfd7803a08c6fac03dd0e2826dc1

**Review by wojcik92michal — APPROVED** [review]

**Review by wercix — APPROVED** [review]

**Pull request merged**